mars/lib/aio/lru.py

```diff
@@ -152,7 +152,7 @@ def _cache_miss(wrapped, key):
 
 def alru_cache(
     fn=None,
-    maxsize=128,
+    maxsize=1024,
```

Comment:
Why not set the maxsize for `WorkerMetaAPI`? The default value 128 is the same with `functools.lru_cache`: https://docs.python.org/3/library/functools.html#functools.lru_cache

Reply:
Agree with you. We can only set cache max size for `WorkerMetaAPI` if 128 is a common value.